query-service/src/main/resources/configs/common/application.conf

```diff
@@ -38,18 +41,201 @@ service.config = {
           viewName = spanEventView
           mapFields = ["tags"]
           fieldMap = {
-            "EVENT.serviceName": "service_name",
-            "EVENT.id": "span_id",
+            "EVENT.serviceId": "service_id",
+            "EVENT.serviceName" : "service_name",
+            "EVENT.apiId" : "api_id",
+            "EVENT.apiName" : "api_name",
+            "EVENT.apiTraceId" : "api_trace_id",
+            "EVENT.id" : "span_id",
             "EVENT.startTime": "start_time_millis",
             "EVENT.endTime": "end_time_millis",
-            "EVENT.traceId": "trace_id",
-            "EVENT.parentSpanId": "parent_span_id",
-            "EVENT.type": "span_kind",
+            "EVENT.traceId" : "trace_id",
+            "EVENT.parentSpanId" : "parent_span_id",
+            "EVENT.type" : "span_kind",
+            "EVENT.entryApiId": "entry_api_id",
+            "EVENT.protocolName": "protocol_name",
             "EVENT.statusCode": "status_code",
-            "EVENT.spanTags": "tags"
-            "EVENT.spanRequestUrl": "request_url",
+            "EVENT.spanTags" : "tags"
+            "EVENT.spanRequestUrl" : "request_url",
             "EVENT.duration": "duration_millis",
+            "EVENT.displayEntityName": "display_entity_name",
             "EVENT.displaySpanName": "display_span_name",
+            "EVENT.errorCount": "error_count",
+            "EVENT.exceptionCount": "exception_count"
+          }
+        }
+      }
+    }
+    {
+      name = service-handler
+      type = pinot
+      clientConfig = zookeeper
+      requestHandlerInfo = {
+        viewDefinition = {
+          viewName = rawServiceView
+          fieldMap = {
+            "SERVICE.startTime": "start_time_millis",
+            "SERVICE.endTime": "end_time_millis",
+            "SERVICE.id": "service_id",
+            "SERVICE.name": "service_name",
+            "SERVICE.hostHeader" : "host_header",
+            "SERVICE.namespace" : "namespace_name",
+            "SERVICE.cluster" : "cluster_name",
+            "SERVICE.duration": "duration_millis",
+            "SERVICE.errorCount": "error_count",
+            "SERVICE.exceptionCount": "exception_count",
+            "SERVICE.numCalls": "num_calls",
+            "API.isExternal" : "api_is_external"
+          }
+        }
+      }
+    }
+    {
+      name = api-traces-view-handler
```

Comment:
Is all handler available in hypertrace-core?

Reply:
@kotharironak  for now we are proceeding with config changes as required for `hypertrace`. 
 pls see slack channel #ht-deploy-launch.

Reply:
No @kotharironak